.github/actions/detect-workflow-js/src/detect.ts

```diff
@@ -46,7 +46,9 @@ export async function detectWorkflowFromOIDC(
     return Promise.reject(Error("job_workflow_ref missing from OIDC token."));
   }
 
-  const [workflowPath, workflowRef] = jobWorkflowRef.split("@", 2);
+  const firstAtIndex = jobWorkflowRef.indexOf("@");
```

Comment:
please add a comment why we need this, with your example `"vitejs/vite/.github/workflows/publish.yml@refs/tags/create-vite@5.0.0-beta.0"`